Gems/PhysX/Code/Source/EditorShapeColliderComponent.cpp

```diff
@@ -440,25 +440,24 @@ namespace PhysX
 
         SetShapeConfig(ShapeType::Capsule, capsuleShapeConfig);
 
-        m_shapeConfigs.back()->m_scale = scale;
-        const float scalarScale = scale.GetMaxElement();
-        m_geometryCache.m_radius = scalarScale * capsuleShapeConfig.m_radius;
-        m_geometryCache.m_height = scalarScale * capsuleShapeConfig.m_height;
+        m_shapeConfigs.back()->m_scale = AZ::Vector3(scale);
```

Comment:
question: Should we check `shapeConfigs` isn't empty just in case?

Reply:
This was always happening immediately after a call to SetShapeConfig, so I've folded the scale change into that function. Inside there, it's guaranteed not to be empty.

Reply:
Ah nice, sounds good thanks!